src/services/userService.js

```diff
@@ -0,0 +1,54 @@
+let Sequelize = require('sequelize');
+
+
+function UserService(model) {
+  this.model = model;
+}
+
+UserService.prototype.createUser = async function(user) {
+  let data = await this.model.create(user);
```

Comment:
Consider wrapping ALL the await logic in your application in try...catch block as you did with "updateUser"
This is a best practice for Node.js async/await - please do this in ALL the files that use async/await of your app.

Reply:
resolved